source/PreferencesPanel.cpp

```diff
@@ -332,7 +333,9 @@ void PreferencesPanel::DrawControls()
 		Command::FIGHT,
 		Command::GATHER,
 		Command::HOLD,
-		Command::AMMO
+		Command::AMMO,
+		Command::NONE,
```

Comment:
This line appears extraneous (although I may be wrong).

Reply:
NONE tells the interface not to draw a box for the category line, it corresponds with the CATEGORIES array just above.